src/main/java/net/apnic/rdap/error/config/ErrorConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package net.apnic.rdap.error.config;
+
+import net.apnic.rdap.error.filters.ZuulErrorFilter;
+import net.apnic.rdap.rdap.RDAPObjectFactory;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ErrorConfiguration
+{
+    @Bean
+    @Autowired
+    public ZuulErrorFilter zuulErrorFilter(RDAPObjectFactory factory)
+    {
+        return new ZuulErrorFilter(factory);
```

Comment:
Google tells me that "zuul" is a genus of dinosaur. I would love for RDAP to have a dinosaur filter, but I suspect that zuul means something different within RDAP. So, what does it mean?